src/redux/sagas/motions/createEditDomainMotion.ts

```diff
@@ -136,36 +264,28 @@ function* createEditDomainMotion({
     //   }),
     // );
 
-    const encodedAction = colonyClient.interface.encodeFunctionData(
-      isCreateDomain
-        ? 'addDomain(uint256,uint256,uint256,string)'
-        : 'editDomain',
-      [permissionDomainId, childSkillIndex, domainId, '.'], // domainMetadataIpfsHash
-    );
-
-    // create transactions
-    yield fork(createTransaction, createMotion.id, {
-      context: ClientType.VotingReputationClient,
-      methodName: 'createMotion',
-      identifier: colonyAddress,
-      params: [
-        motionDomainId,
-        motionChildSkillIndex,
-        AddressZero,
-        encodedAction,
-        key,
-        value,
-        branchMask,
-        siblings,
-      ],
-      group: {
-        key: batchKey,
-        id: metaId,
+    const transactionParams = yield call(getCreateTransactionPayload, {
+      base: {
```

Comment:
yeah looking at this, I think we can safely send in just one single object instead of a split here

Reply:
Yeah makes sense now since I don't have any unique params set to use for multi-sig